### PR TITLE
Javascript javascript2 week2

### PR DESCRIPTION
**Pull request by melo4yxa77796**
my home work for this week

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -32,47 +32,56 @@ const emojis = [
...
-    });
+document.addEventListener("DOMContentLoaded", (event) => {
+  const startButton = document.getElementById("start");
+  startButton.addEventListener("click", () => {
+    window.location.reload();
+  });
 });
 
-let shuf_emojis = emojis.sort(function () {
+/*let shuf_emojis = emojis.sort(function () {
```
Comment (vdaguenet) [inline index.js:42]: Avoid leaving dead code in your PR

File: index.js [hunk 2]
```diff
@@ -32,47 +32,56 @@ const emojis = [
...
   return Math.random() > 0.5 ? 1 : -1;
-});
+});*/
+const gameContainer = document.querySelector(".game-container");
+let shuf_emojis = emojis.sort(() => (Math.random() > 0.5 ? 1 : -1));
```
Comment (vdaguenet) [inline index.js:46]: Nice!

**Review by vdaguenet — APPROVED** [review]
Very nice work!

File: index.js (hunk 1 above)
Comment (melo4yxa77796) [reply index.js:42]: okay

**Pull request merged**